newton/tests/test_mujoco_solver.py

```
@@ -4965,6 +4965,51 @@ def test_custom_attributes_from_usd(self):
         assert np.allclose(model.mujoco.condim.numpy(), [6])
         assert np.allclose(solver.mjw_model.geom_condim.numpy(), [6])
 
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_mjc_damping_from_usd_via_schema_resolver(self):
+        """Test mjc:damping attributes are parsed via SchemaResolverMjc."""
+        from pxr import Sdf, Usd, UsdGeom, UsdPhysics  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)